pylti1p3/exception.py

```diff
@@ -1,6 +1,33 @@
+import typing as t
+
+if t.TYPE_CHECKING:
+    # pylint: disable=unused-import
+    import requests
+
+
 class LtiException(Exception):
     pass
 
 
 class OIDCException(Exception):
     pass
+
+
+class LtiServiceException(LtiException):
+    def __init__(self, response):
```

Comment:
hi @libre-man . I agree to add HTTP response into `LtiServiceException` but i don't like idea of `maybe_raise` method. I suggest to remove it and check status code as it is done now:
```
if not response.ok:
    raise LtiServiceException(response)
```